include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -39,223 +39,204 @@
 
 #elif BOOST_OS_CYGWIN
-                // New 64-bit MEMORYSTATUSEX isn't available.
-                MEMORYSTATUS status;
-                status.dwLength = sizeof(status);
-                GlobalMemoryStatus(&status);
-                return static_cast<std::size_t>(status.dwTotalPhys);
+        // New 64-bit MEMORYSTATUSEX isn't available.
+        MEMORYSTATUS status;
+        status.dwLength = sizeof(status);
+        GlobalMemoryStatus(&status);
+        return static_cast<std::size_t>(status.dwTotalPhys);
 
 #elif BOOST_OS_UNIX || BOOST_OS_MACOS
-                // Unix : Prefer sysctl() over sysconf() except sysctl() with HW_REALMEM and HW_PHYSMEM which are not
-                // always reliable
+        // Unix : Prefer sysctl() over sysconf() except sysctl() with HW_REALMEM and HW_PHYSMEM which are not
+        // always reliable
 #    if defined(CTL_HW) && (defined(HW_MEMSIZE) || defined(HW_PHYSMEM64))
-                int mib[2]
-                    = { CTL_HW,
+        int mib[2]
+            = { CTL_HW,
 #        if defined(HW_MEMSIZE) // OSX
-                        HW_MEMSIZE
+                HW_MEMSIZE
 #        elif defined(HW_PHYSMEM64) // NetBSD, OpenBSD.
-                        HW_PHYSMEM64
+                HW_PHYSMEM64
 #        endif
-                      };
-                std::uint64_t size(0);
-                std::size_t sizeLen{sizeof(size)};
-                if(sysctl(mib, 2, &size, &sizeLen, nullptr, 0) < 0)
-                    throw std::logic_error("getTotalGlobalMemSizeBytes failed calling sysctl!");
-                return static_cast<std::size_t>(size);
+              };
+        std::uint64_t size(0);
+        std::size_t sizeLen{sizeof(size)};
+        if(sysctl(mib, 2, &size, &sizeLen, nullptr, 0) < 0)
+            throw std::logic_error("getTotalGlobalMemSizeBytes failed calling sysctl!");
+        return static_cast<std::size_t>(size);
 
 #    elif defined(_SC_AIX_REALMEM) // AIX.
-                return static_cast<std::size_t>(sysconf(_SC_AIX_REALMEM)) * static_cast<std::size_t>(1024);
+        return static_cast<std::size_t>(sysconf(_SC_AIX_REALMEM)) * static_cast<std::size_t>(1024);
 
 #    elif defined(_SC_PHYS_PAGES) // Linux, FreeBSD, OpenBSD, Solaris.
-                return static_cast<std::size_t>(sysconf(_SC_PHYS_PAGES)) * getPageSize();
+        return static_cast<std::size_t>(sysconf(_SC_PHYS_PAGES)) * getPageSize();
 
 #    elif defined(CTL_HW)                                                                                             \
         && (defined(HW_PHYSMEM) || defined(HW_REALMEM)) // FreeBSD, DragonFly BSD, NetBSD, OpenBSD, and OSX.
-                int mib[2]
-                    = { CTL_HW,
+        int mib[2]
+            = { CTL_HW,
 #        if defined(HW_REALMEM) // FreeBSD.
-                        HW_REALMEM;
+                HW_REALMEM
 #        elif defined(HW_PYSMEM) // Others.
-                        HW_PHYSMEM;
+                HW_PHYSMEM
```

Comment:
I think the extra `;` at the end of these lines were actually a bug (and anyway they confused clang-format a lot).